src/api/domain/exchanges-api.ts

```diff
@@ -617,7 +627,7 @@ export class ExchangesApi extends BaseAPI {
      * @memberof ExchangesApi
      */
     public createExchangeTransfer(requestParameters: ExchangesApiCreateExchangeTransferRequest, options?: AxiosRequestConfig) {
-        return ExchangesApiFp(this.configuration).createExchangeTransfer(requestParameters.id, requestParameters.xImxEthAddress, requestParameters.xImxEthSignature, requestParameters.createTransferRequest, options).then((request) => request(this.axios, this.basePath));
+        return ExchangesApiFp(this.configuration).createExchangeTransfer(requestParameters.id, requestParameters.createTransferRequest, requestParameters.xImxEthAddress, requestParameters.xImxEthSignature, requestParameters.authorization, options).then((request) => request(this.axios, this.basePath));
```

Comment:
@Nathan-Ballantyne this is the actual entry point, which uses the object for parameters

Reply:
I see! Thanks this looks good to me!